### PR TITLE
[WebInterface] community tentacles

### PR DESCRIPTION
**Pull request by GuillaumeDSM**
requires https://github.com/Drakkar-Software/OctoBot/pull/1679
image
image
image
image
image

### CODE REVIEW

File: Services/Interfaces/web_interface/static/js/components/community.js [hunk 1]
```diff
@@ -16,9 +16,59 @@
...
+function reloadOwnedPackages(){
+    $("#owned-tentacles").load(location.href + " #owned-tentacles", function(){
+        reloadTable();
+    });
+}
+
+function packagesOperationSuccessCallback(updated_data, update_url, dom_root_element, msg, status){
+    disablePackagesOperations(false);
+    reloadOwnedPackages();
+    create_alert("success", "Packages operation success", msg);
```
Comment (Herklos) [inline Services/Interfaces/web_interface/static/js/components/community.js:60]: You meant `succeed`, didn't you?

**Comment by Herklos** [comment]
I don't think tab "real trading OctoBots this month" (and the two others) color really matches the theme.

File: Services/Interfaces/web_interface/static/js/components/community.js (hunk 1 above)
Comment (GuillaumeDSM) [reply Services/Interfaces/web_interface/static/js/components/community.js:60]: `success` is a keyword to create a "green" alert on the web interface

Comment (Herklos) [reply Services/Interfaces/web_interface/static/js/components/community.js:60]: I was talking about `"Packages operation success"`

Comment (GuillaumeDSM) [reply Services/Interfaces/web_interface/static/js/components/community.js:60]: Oh, right 👍

**Review by Herklos — APPROVED** [review]

**Pull request merged**